src/main/javac/org/apache/brooklyn/tosca/a4c/platform/transformer/ToscaPlanToSpecTransformer.java

```diff
@@ -224,9 +239,27 @@ private static boolean isTosca(Map<?,?> obj) {
 
```

Comment:
I suppose that you want to use custom NodeTypes which represent the Brooklyn entities. For example
```
node_types:
  org.apache.brooklyn.entity.webapp.tomcat.TomcatServer:
    derived_from: tosca.nodes.WebServer
    description: >
      A simple Tomcat server
    properties:
      wars.root:
        type: string
        required: false
      wars.named:
        type: string
        required: false
      http.port:
        type: list
        required: false
        entry_schema:
          type: string
      host:
        type: string
        required: true
```
Probably the last NodeType should extend `  org.apache.brooklyn.entity.webapp.JavaWebAppSoftwareProcess NodeType`.

Using these custom types we will avoid create a converter for each NodeType (like Camp support does currently).